src/SignalR/clients/java/signalr/signalr.client.java.Tests.javaproj

```diff
@@ -1,5 +1,5 @@
-<Project DefaultTargets="Build">
```

Comment:
The project rename to have `.Tests` at the end is an unfortunate side-effect of how arcade determines if a project is a test or not, which we use when deciding how helix should behave.